### DISCUSSION TITLE
Graphic design

### DISCUSSION THREAD

**Discussion by Bitashf** [WeAreMahsaAmini/Core#323] [Introductions]
Hi guys, I'm a graphic designer, how can I help? I work with Photoshop, InDesign and illustrator

**Comment by ghost** [comment]
Hi, I'm a graphic designer too, how can I help?

**Reply by FB-888222** [reply]
سلام دوستان
حامد اسماعیلیون و تیمش برای فراخوان تظاهرات مردمی برای هرشهر یک پوستر طراحی کردند و تاریخ و ساعت و مکان تجمعات رو نوشتن که برای روز شنبه ساعت ۱۲:۳۰ هست.
الان توی خیلی از شهر های ایران مردم بیرون میرن ولی چون فراخوان وجود نداره و محله محور نیست اعتراضات خیلی نتیجه نمیده.
پیشنهاد من این بود که دوستانی که میتونن مشابه پوستر های اقای اسماعیلیون برای شهر های ایران طراحی بشه و هم چنین نام دو یا سه محله از شهر های بزرگ نوشته بشه که مردم بدونن محل تقریبی کجاست.
ممنون از پروژه و کمک هاتون
#مهسا_امینی

**Comment by FB-888222** [comment]
سلام دوستان
حامد اسماعیلیون و تیمش برای فراخوان تظاهرات مردمی برای هرشهر یک پوستر طراحی کردند و تاریخ و ساعت و مکان تجمعات رو نوشتن که برای روز شنبه ساعت ۱۲:۳۰ هست.
الان توی خیلی از شهر های ایران مردم بیرون میرن ولی چون فراخوان وجود نداره و محله محور نیست اعتراضات خیلی نتیجه نمیده.
پیشنهاد من این بود که دوستانی که میتونن مشابه پوستر های اقای اسماعیلیون برای شهر های ایران طراحی بشه و هم چنین نام دو یا سه محله از شهر های بزرگ نوشته بشه که مردم بدونن محل تقریبی کجاست.
ممنون از پروژه و کمک هاتون
#مهسا_امینی

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر

**Comment by parhamsms** [comment]
@Bitashf we need you for two projects. Search **Medical support** and **flag Iran**

**Comment by karimifar** [comment]
@Bitashf There is an open invitation for artists and designers here: https://drive.google.com/drive/u/2/folders/1DgRBz8EFMHHuo4SXqeZaeOsgCKTvewIo
